schema.graphql

```diff
@@ -115,51 +162,98 @@ type DecimalValue @entity {
   decimals: Int!
 }
 
-type ValoremDayData @entity {
-  # timestamp rounded to current day by dividing by 86400
+type ClearinghouseDayData @entity {
```

Comment:
Let's just call it DayData

Reply:
:+1: 
done https://github.com/valorem-labs-inc/valorem-subgraph/pull/43/commits/f1be2e1a5d3eac6e222d336be498649b79daacea